MetasysServicesCom/Interfaces/ILegacyMetasysClient.cs

```diff
@@ -260,6 +261,13 @@ public interface ILegacyMetasysClient
         /// <returns></returns>
         void DiscardAudit(string id, string annontationText);
 
+        /// <summary>
+        /// Discard many Audit given a list of requests containing the Id of the Audits and the text for the Annotations.
+        /// </summary>
+        /// <param name="requests">List of BatchRequestParam to specify the id of the audits and the text of the annotations to discard.</param>
+        /// <returns>A list of BatchRequestParam with all the specified attributes.</returns>
+        string[] DiscardAuditMultiple(string[] requests);

```

Comment:
The 'MarshalAs' decorator is needed otherwise will cause a VBA error. 
Please modify the parameter as 
'([In, MarshalAs(UnmanagedType.LPArray, SizeParamIndex = 0)] string[] requests)'